framework/auth/views.py

```diff
@@ -130,17 +132,18 @@ def auth_login(auth, **kwargs):
     if status_message == 'expired':
         status.push_status_message('The private link you used is expired.')
 
-    code = http.OK
     if next_url:
         status.push_status_message(language.MUST_LOGIN)
-        # Don't raise error if user is being logged out
-        if not request.args.get('logout'):
```

Comment:
I admit it's a bit odd to call the login url with a `logout` parameter- but was this deletion intentional? (was this meant to cover some sort of edge case?)